App.js

```diff
@@ -8,16 +9,23 @@ import {TabNavigator, StackNavigator} from 'react-navigation';
 import {FIREBASE_CONFIG} from './env';
 import reducers from './src/reducers';
 
+import {navKeys} from './src/constants';
```

Comment:
I refactored the hard-coded strings we had for the navigation keys, such as `login`, `filter`, `search` , etc. and placed them all into the `constants.js` file. We use them in so many places, we don't want to have to hunt them all down any time we refactor anything.